src/webpack/webpackTools.ts

```diff
@@ -48,9 +59,38 @@ export function createConfig(outPath: string, entryPoint: string | webpack.Entry
         },
         module: {
             rules: [
-                {test: /\.css$/, loader: 'style-loader'},
-                {test: /\.css$/, loader: 'css-loader'},
-                {test: /\.module.css$/, loader: 'css-loader', query: {modules: true, localIdentName: '[name]__[local]___[hash:base64:5]'}},
+                {
+                    test: /\.module\.(sc|sa|c)ss$/,
+                    loader: [
+                        finalCSSLoader(extractCSS),
```

Comment:
Change `*CSS` to `*Css`? Think we're more often using `Url` etc. in code, and consistent with `MiniCssExtractPlugin`

Reply:
Same change for `extractCSS`

Reply:
Renamed.